db/schema.rb

```diff
@@ -129,22 +122,19 @@
     t.datetime "confirmed_at"
     t.datetime "confirmation_sent_at"
     t.string "unconfirmed_email"
-    t.string "first_name"
-    t.string "username"
+    t.string "name"
```

Comment:
uhh, okay, I guess it'll fix in the migrations

Reply:
i'll make a Migration for this and last_name and description, which are apparently missing. @NicholasLYang schema may be the reason why I had those weird errors with updating user descriptions and last names.